src/py/flwr/server/superlink/fleet/vce/vce_api.py

```diff
@@ -87,33 +86,32 @@ def _register_node_info_stores(
 
 # pylint: disable=too-many-arguments,too-many-locals
 def worker(
-    taskins_queue: "Queue[TaskIns]",
-    taskres_queue: "Queue[TaskRes]",
+    messageins_queue: "Queue[Message]",
+    messageres_queue: "Queue[Message]",
```

Comment:
Do we need the double quotes here?

```suggestion
    messageins_queue: Queue[Message],
    messageres_queue: Queue[Message],
```

Reply:
for python3.9 only (the alternative would be to import Queue from `typing` instead for py39)